src/benchmark/tpcc_benchmark.cpp

```diff
@@ -108,8 +108,8 @@ void check_consistency(const size_t num_warehouses) {
     Assert(table && table->row_count() == static_cast<size_t>(num_warehouses), "Lost a warehouse");
     const auto row_count = table->row_count();
     for (auto row_id = size_t{0}; row_id < row_count; ++row_id) {
-      const auto w_ytd = double{table->get_value<float>(ColumnID{1}, row_id)};
-      const auto d_ytd_sum = table->get_value<double>(ColumnID{2}, row_id);
+      const auto w_ytd = double{*table->get_value<float>(ColumnID{1}, row_id)};
```

Comment:
You'll see a couple of these that are caused by `get_value` now returning an optional.